### ISSUE TITLE
zaplocker.com site is offline

### ISSUE THREAD

**Issue by YuckFouBTC** [supertestnet/zaplocker#8]
zaplocker.com is down.  See [https://archive.ph/AylLM](https://archive.ph/AylLM)

**Comment by supertestnet** [comment]
Yes, I was self hosting the server at Pleblab but we moved to a new building that does not have a static IP address. I can't get the site up til we get one, which isn't really a priority. Consider it down indefinitely.

**Comment by justingoldberg** [comment]
Wen bitdns?

Nostr based dyndns?

https://nomenexplorer.com/
